sound/soc/sof/intel/hda.h

```diff
@@ -408,6 +408,7 @@ static inline struct hda_bus *sof_to_hbus(struct snd_sof_dev *s)
 }
 
 struct sof_intel_hda_stream {
+	struct snd_sof_dev *sdev;
```

Comment:
might be worth adding a sdev_to_hda_stream() macro to avoid the repeated container_of in future patches?

Reply:
@plbossart I am not sure we need this macro. I only use the sdev when I have the sof_intel_hda_stream pointer and want to access the sdev. But never the other way around.